tony-core/src/main/java/com/linkedin/tony/TonyApplicationMaster.java

```diff
@@ -124,7 +151,7 @@
   private TonySession.Builder sessionBuilder;
 
   /** Configuration **/
-  private Configuration yarnConf;
+  private static Configuration yarnConf;
```

Comment:
what?

Reply:
?

Reply:
meant why static

Reply:
Once I do https://github.com/linkedin/TonY/pull/79#discussion_r233594390, then this shouldn't be static.